man/rmd-fragments/format-latex.Rmd

```diff
@@ -28,3 +28,22 @@ you must be sure to either use punctuation after the trailing dollar sign OR for
     math$head(15) # malformed
     math$protect_math() #error
     ```
+4. **use of `$` as currency will still work**, but there is a caveat that mixing
```

Comment:
why the two *?

I'd break this sentence in two. Saying it works, saying "nevertheless, if there's broken math (<what is broken math>)..."